content/2021/09/demyst-offers-data-from-minfraud.mdx

```diff
@@ -1,28 +1,48 @@
 ---
 title: "DemystData’s fraud solutions offer best-in-class IP risk data from the minFraud service to help financial services firms prevent fraud"
 date: "2021-09-07"
-categories: 
+categories:
   - "featured-entry-2"
   - "online-fraud-prevention"
-tags: 
+tags:
   - "financial-services"
   - "fraud-screening"
   - "minfraud"
   - "online-fraud-detection"
 ---
 
-[Demyst](https://demyst.com/)’s Platform has helped Tier 1 financial services firms modernize and leverage external data workflows for over 10 years. Account takeover and synthetic fraud can be expensive for Demyst’s clients in online payments, banking, and insurance. Demyst’s VP of Product Strategy Prashant Reddy explains that the data provided by the minFraud service fills a key niche in the solutions Demyst’s clients are creating.
+[Demyst](https://demyst.com/)’s Platform has helped Tier 1 financial services
+firms modernize and leverage external data workflows for over 10 years. Account
+takeover and synthetic fraud can be expensive for Demyst’s clients in online
+payments, banking, and insurance. Demyst’s VP of Product Strategy Prashant Reddy
+explains that the data provided by the minFraud service fills a key niche in the
+solutions Demyst’s clients are creating.
 
-### “minFraud’s IP scoring provides value to our clients, but some clients also get value from additional checks. Does the location of the device conducting a transaction match the consumer’s reported location? Is this a high risk mailing address? This is extremely useful information for screening transactions or deciding whether to do manual review.”
+### “minFraud’s IP scoring provides value to our clients, but some clients also
```

Comment:
I think this should be a block quote rather than a header. However, if it is preferred to stay a header, only part of the quote is currently a heading. (+Same comment for other areas of this file)